core/web/resolver/resolver_test.go

```diff
@@ -173,7 +173,7 @@ func RunGQLTests(t *testing.T, testCases []GQLTestCase) {
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			t.Parallel()
+			//t.Parallel()
```

Comment:
Restore?

Reply:
🤦  Restored in https://github.com/smartcontractkit/chainlink/pull/11386/commits/3ee7f3042da13414a491a88f3bb4e6083379bfd2